datafusion-cli/tests/cli_integration.rs

```diff
@@ -74,6 +75,31 @@ fn cli_quick_test<'a>(
     assert_cmd_snapshot!(cmd);
 }
 
+#[rstest]
```

Comment:
I could not get these to work as additional examples in `quick_test` for reasons I couldn't figure out. So I just made a new test

Reply:
Thanks, pulled into this branch